src/serialize/ready.rs

```diff
@@ -0,0 +1,60 @@
+//! Message sent from datapath to CCP when it starts up, indicating its address
```

Comment:
Do we need corresponding changes in libccp?

Reply:
Yep, see [this commit](https://github.com/ccp-project/libccp/commit/b599a0361895966fb09aaf647aad120ae7a3342d
)